pkg/internal/appolly/appolly.go

```diff
@@ -95,7 +87,7 @@ func (i *Instrumenter) FindAndInstrument(wg *sync.WaitGroup) error {
 		}
 	}()
 	// TODO: wait until all the resources have been freed/unmounted
```

Comment:
Not sure if this already happen. I will try to check/fix the RingbufForwarder in a future PR.